src/plugins/home/public/application/kibana_services.ts

```diff
@@ -52,7 +54,11 @@ export interface HomeKibanaServices {
   addDataService: AddDataService;
   welcomeService: WelcomeService;
   guidedOnboardingService?: GuidedOnboardingApi;
-  cloud?: CloudSetup;
+  cloud: CloudSetup;
+  openModal: OverlayStart['openModal'];
+  theme: ThemeServiceStart;
+  i18nStart: I18nStart;
+  shareStart: SharePluginStart;
```

Comment:
```suggestion
  share: SharePluginStart;
```

Reply:
the name share is already used for the share setup in the getting_started.tsx file T_T